slides/k8s/localkubeconfig.md

```diff
@@ -192,4 +192,4 @@ class: extra-details
 
 ]
 
-We can now utilize the cluster exactly as we did before, except that it's remote.
+We can now utilize the cluster exactly as if we're logged into a node, except that it's remote.
```

Comment:
Shouldn't that be "as if we were"?

Reply:
If you like. I was going with the informality of present tense/indicative mood, but if you prefer the contrary-to-fact conditional, then the subjunctive mood is fine. Both are technically correct English; mine is less formal.

Reply:
OK, thanks for the explanation! I'll merge that now.